spec/models/block_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Block do
+  describe '#current?' do
+    let!(:user) { create(:user) }
+    let!(:block) { create(:block, user: user) }
+
+    it 'true' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.